services/nginx/nginx.conf

```diff
@@ -0,0 +1,40 @@
+error_log /dev/stderr;
+daemon off;
+pid /tmp/nginx_pid;
+
+# Load the pam module for authentication
+load_module /usr/lib/nginx/modules/ngx_http_auth_pam_module.so;
+
+events {}
+
+http {
+    include /etc/nginx/mime.types;
+    access_log /dev/stderr;
+
+    upstream discourse_sso {
+        server 127.0.0.1:8000;
+    }
+
+    server {
+        listen 8080;
+
+        client_body_temp_path /tmp/nginx_client_temp;
```

Comment:
are any of these directives actually necessary? same with the `events {}` block and other things

Reply:
Yes, omitting any of these causes nginx to error out when loading.